### DISCUSSION TITLE
How can Picker get path rather than uri on Android platform?

### DISCUSSION THREAD

**Discussion by DJDQfff** [unoplatform/uno#15054] [Q&A]
On Android platforms,when I use FileOpenPicker/FolderPicker  to picka storageitem, the Path property value is a URI rather tahn path.

For example, the file is "/emulated/0/test/1.txt".

If I use FileOpenPicker, StorageFile.Path value is "content://com.android.externalstorage.documents/document/primary%3Atest%2Fqqimage.png".What I hope  is "/emulated/0/test/1.txt".

If I use FolderPicker to pick the test folder, StorageFolder.Path value is "/tree/primary:test".

How can I get its path ?

**Comment by MartinZikmund** [comment]
I think this is not possible due to the sandboxed nature of file management on Android. I will check though to verify
